IFC4x3/Properties/n/NominalDepth_0HZnE0qUaHuO00025QrE$V/DocProperty.xml

```diff
@@ -7,3 +7,4 @@
 		</DocLocalization>
 	</Localization>
 </DocProperty>
+
```

Comment:
This is a weird empty line that snuck in. @jmirtsch do you think this is a bug?

Reply:
Weird...